src/Plugin.php

```diff
@@ -79,6 +81,7 @@ public static function init() {
         'menu_title' => __('Hide "Add to Cart" button', Plugin::L10N),
         'parent_slug' => 'woocommerce',
       ]);
+      ProductAttributePageTitle::init();
```

Comment:
I think you can just add a WP hook to invoke this, similar to how Plugin::init() is called here: https://github.com/netzstrategen/wordpress-shop-standards/blob/2c9771e76c06076b917fcd08353c70310649dbc0/plugin.php#L42

Perhaps on the next line you can just insert the following

```php
add_action('init', __NAMESPACE__ . '\ProductAttributePageTitle::init', 20);
```

Reply:
@wrux These are wordpress tags not woocommerce attribute/term settings fields.